_includes/header.html

```diff
@@ -1,35 +1,58 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-primary">
-  <div class="container">
-    <a class="navbar-brand text-success" rel="author" href="{{ "/" | relative_url }}">{{ site.title | escape }}</a>
+<header class="flex-shrink-0">
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+    <div class="container">
+      <a href="{{ "/" | relative_url }}" class="navbar-brand" rel="author">
+        <img src="/assets/Hacker101_logo.png" alt="View Hacker101 homepage" height="30">
+      </a>
 
-    {%- assign page_paths = site.header_pages -%}
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
-      aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
-      <ul class="navbar-nav mr-auto">
-        <li class="nav-item">
-            <a class="nav-link" href="{{ "/start-here" | relative_url }}">Start Here</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="{{ "/videos" | relative_url }}">Videos</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="https://ctf.hacker101.com/">CTF</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="{{ "/resources" | relative_url }}">Resources</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="{{ "/discord" | relative_url }}">Discord</a>
-        </li>
-      </ul>
-      <ul class="navbar-nav ml-auto">
-        <li class="nav-item">
-          <a class="nav-link" href="{{ site.github.repository_url }}">Fork me on GitHub</a>
-        </li>
-      </ul>
+      {%- assign page_paths = site.header_pages -%}
+
+      <button class="navbar-toggler p-0 border-0" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <!-- <li class="nav-item"> -->
```

Comment:
Is this needed?

Reply:
Probably not, originally was going to add another page, but ran out of time to do so.